webtreemap.js

```diff
@@ -12,11 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+;(function(root, factory) {
+  if (typeof define === 'function' && define.amd)
+    define([], factory);
+  else if (typeof module === 'object' && module.exports)
+    module.exports = factory();
+  else
+    root.appendTreemap = factory();
+}(this, function() {
```

Comment:
Is this really required?  Why don't I see this in other JS source bases?  (Sorry, I'm pretty unfamiliar with all the modern trends in this stuff.)

Isn't this an API change, in that window.appendTreemap disappears after your change?